.github/workflows/simulation.yml

```diff
@@ -1,436 +0,0 @@
-#
-#  Copyright (c) 2020, The OpenThread Authors.
-#  All rights reserved.
-#
-#  Redistribution and use in source and binary forms, with or without
-#  modification, are permitted provided that the following conditions are met:
-#  1. Redistributions of source code must retain the above copyright
-#     notice, this list of conditions and the following disclaimer.
-#  2. Redistributions in binary form must reproduce the above copyright
-#     notice, this list of conditions and the following disclaimer in the
-#     documentation and/or other materials provided with the distribution.
-#  3. Neither the name of the copyright holder nor the
-#     names of its contributors may be used to endorse or promote products
-#     derived from this software without specific prior written permission.
-#
-#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
-#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
-#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-#  POSSIBILITY OF SUCH DAMAGE.
-#
-
-name: Simulation
```

Comment:
What would happen to the badge icon: ![Simulation](https://github.com/openthread/openthread/workflows/Simulation/badge.svg) in the README.md

Reply:
Ideally I think we want it to indicate aggregation of all `Build` related workflows (show it `passing` if al of them arel passing). I don't know if this is possible or not.

Thoughts? Suggestions?

Reply:
I don't think GItHub Actions provides an ability to aggregate status into a single badge.

Reply:
I could not find any immediate solution to enable aggression 

We can consider including badges for a subset of the workflows in `README.md` file?

Thoughts?

Reply:
My suggestion would be:

![Build (arm-gcc)](https://github.com/openthread/openthread/workflows/Build%20(arm-gcc)/badge.svg) ![Build (gcc)](https://github.com/openthread/openthread/workflows/Build%20(gcc)/badge.svg) ![Build (clang)](https://github.com/openthread/openthread/workflows/Build%20(clang)/badge.svg) ![Build (macos)](https://github.com/openthread/openthread/workflows/Build%20(macos)/badge.svg) ![Simulation (cli)](https://github.com/openthread/openthread/workflows/Simulation%20(cli)/badge.svg) ![Simulation (cli)](https://github.com/openthread/openthread/workflows/Simulation%20(ncp)/badge.svg) ![Simulation 1.2](https://github.com/openthread/openthread/workflows/Simulation%201.2/badge.svg) ![POSIX](https://github.com/openthread/openthread/workflows/POSIX/badge.svg) ![Docker](https://github.com/openthread/openthread/workflows/Docker/badge.svg)

Is it too many ?